stories/topics/proc-aap-aws-backup-and-recovery.adoc

```diff
@@ -1,110 +1,191 @@
+-e AWS_ACCESS_KEY_ID=<Your AWS Access Key ID>
+-e AWS_SECRET_ACCESS_KEY=<Your AWS Secret Access Key>
+$IMAGE \
+redhat.ansible_on_clouds.aws_backup_stack \
+-e @/extra_vars/extra_vars.yaml
+----
+
+.. Replace `<absolute path of extra vars dir>` with the path to your directory where your `extra_vars.yaml` file exists which you have created in step 1 of the backup process and replace `<Your AWS Access Key ID>` and `<Your AWS Secret Access Key>` with the values for the AWS account you want to run the backup with. For more information on finding your AWS account credentials see link:https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-about[Accessing AWS using your AWS credentials].
+
+.. Note the output of the playbook.
++
+The playbook will return something like this
++
+[source, bash]
+----
+{
+    "efs_restore_points": [
+        {
+            "CreationDate": "2023-02-28T21:54:07.515000+00:00",
+            "RecoveryPointArn": "arn:aws:backup:us-east-1:080914983077:recovery-point:967be465-a43e-432a-b536-8b16e6b0452d"
+        },
+        {
+            "CreationDate": "2023-02-28T16:08:08.905000+00:00",
+            "RecoveryPointArn": "arn:aws:backup:us-east-1:080914983077:recovery-point:d2058e3a-dca6-4a18-b175-5ef624f513a7"
+        }
+    ],
+    "rds_db_snapshot_arn": "arn:aws:rds:us-east-1:080914983077:snapshot:test-bnr-ops-container-rds169785b9-orm2iuzlfqem-snap-2023-02-28"
+}
+----
++
+The playbook will have successfully created an EFS backup and RDS snapshot of your deployment listing the most recent EFS restore point and RDS snapshot information. Make a note of the `rds_db_snapshot_arn` and the latest `RecoveryPointArn`. These will be needed for the restore playbook.
++
+
+
+== Recovery Process
 
 .Procedure
-. In the AWS Console, navigate to menu:CloudFormation[].
-. Click on btn:[Create Stack] near the top right of the screen.
-. Select *With new resources (standard)* in the dropdown.
-. In the *Specify template* section under *Template source* click btn:[Upload a template file] and select the template you have modified.
-. Click btn:[Next] to go to the next step
-. Enter a stack name and select an AWSKeyPair to use for the deployment.
-. Click btn:[Next] to go to the *Configure stack options* page.
-. Optional: You can configure stack options and click btn:[Next] to review the stack
-. Click btn:[Submit] to create your new deployment using an RDS Snapshot and restored EFS.
-
-You can now log in succesfully to {PlatformName} {ControllerName} and {HubName} using your old deployment credentials. 
-In addition, all job history, uploaded collections and other records should be in the same state as the restored deployment.
\ No newline at end of file
+. Gather the following input variables to create an `extra_vars.yaml` file like below
++
+[source, yaml]
+----
+aws_foundation_stack_name: "my-foundation-stack"
+
+aws_restored_stack_name: "my-foundation-stack-restored"
```

Comment:
@prat98: the `aws_restored_stack_name` cannot be same as `aws_foundation_stack_name` which must be noted as an assumption in the docs